### PR TITLE
515.43.04 support

### PR DESCRIPTION
**Pull request by Shvarpa**
**Purpose of proposed changes**
adding support for driver 515.43.04, same as 510.68.02

**Essential steps taken**
tested nvidia-fbc patch and it seems to work

### CODE REVIEW

File: patch.sh [hunk 1]
```diff
@@ -181,6 +181,7 @@ declare -A patch_list=(
     ["510.54"]='s/\xe8\x15\x1f\xff\xff\x85\xc0\x41\x89\xc4/\xe8\x15\x1f\xff\xff\x29\xc0\x41\x89\xc4/g'
     ["510.60.02"]='s/\xe8\x15\x1f\xff\xff\x85\xc0\x41\x89\xc4/\xe8\x15\x1f\xff\xff\x29\xc0\x41\x89\xc4/g'
     ["510.68.02"]='s/\xe8\x15\x1f\xff\xff\x85\xc0\x41\x89\xc4/\xe8\x15\x1f\xff\xff\x29\xc0\x41\x89\xc4/g'
+    ["515.43.04"]='s/\xe8\x15\x1f\xff\xff\x85\xc0\x41\x89\xc4/\xe8\x15\x1f\xff\xff\x29\xc0\x41\x89\xc4/g'
```
Comment (jailuthra) [inline patch.sh:184]: The bytecode for NVENC has changed in this release - and according to #528 your patch doesn't fix the NVENC limit.

Can you remove the NVENC changes from your PR? I'll reverse the new binary and handle it separately.

File: README.md [hunk 2]
```diff
@@ -162,6 +162,7 @@ git clone https://ipfs.io/ipns/Qmed4r8yrBP162WK1ybd1DJWhLUi4t6mGuBoB9fLtjxR7u nv
 | 510.54 | YES | YES | [Driver link](https://international.download.nvidia.com/XFree86/Linux-x86_64/510.54/NVIDIA-Linux-x86_64-510.54.run) |
 | 510.60.02 | YES | YES | [Driver link](https://international.download.nvidia.com/XFree86/Linux-x86_64/510.60.02/NVIDIA-Linux-x86_64-510.60.02.run) |
 | 510.68.02 | YES | YES | [Driver link](https://international.download.nvidia.com/XFree86/Linux-x86_64/510.68.02/NVIDIA-Linux-x86_64-510.68.02.run) |
+| 515.43.04 | YES | YES | [Driver link](https://international.download.nvidia.com/XFree86/Linux-x86_64/515.43.04/NVIDIA-Linux-x86_64-515.43.04.run) |
```
Comment (jailuthra) [inline README.md:165]: You don't have to directly edit the README. Instead add the new patch metadata in [drivers.json](https://github.com/keylase/nvidia-patch/blob/master/drivers.json) file, and use`tools/readme-autogen/readme_autogen.py` script to automatically generate the README.

**Review by jailuthra — CHANGES_REQUESTED** [review]
Thanks for the contribution, please fix the minor issues before I merge!

**Comment by jailuthra** [comment]
Will merge this as part of #529

**Comment by Shvarpa** [comment]
🤙

**Pull request closed without merging**